mod/consensus/pkg/cometbft/cli/commands.go

```diff
@@ -252,7 +252,13 @@ func BootstrapStateCmd[T types.Application](
 					return err
 				}
 
-				app := appCreator(logger, db, nil, serverCtx.Viper)
+				app := appCreator(
+					logger,
+					db,
+					nil,
+					serverCtx.Config,
+					serverCtx.Viper,
```

Comment:
we could use client.GetViperFromCmd() and client.GetConfigFromCmd here

https://github.com/cosmos/cosmos-sdk/pull/20422/files